README.md

```diff
@@ -52,21 +53,22 @@ automatically removed from `production` builds:
 </article>
 ```
 
-Once you've done that you can use the `testSelector()` function to create
+Once you've done that you can use the `ts` template string handler to create
 a CSS/jQuery selector that looks up the right elements:
 
 ```js
-import testSelector from 'ember-test-selectors';
+import { ts } from 'ember-test-selectors';
 
 // in Acceptance Tests:
 
-find(testSelector('post-title')) // => find('[data-test-post-title]')
-find(testSelector('resource-id', '2')) // => find('[data-test-resource-id="2"]')
+find(ts`%post-title`); // => find('[data-test-post-title]');
+find(ts`%resource-id=2`); // => find('[data-test-resource-id="2"]');
+find(ts`.blog %post-title input`); // => find('.blog [data-test-post-title] input');
 
 // in Component Integration Tests:
 
-this.$(testSelector('post-title')).click() // => this.$('[data-test-post-title]').click()
-this.$(testSelector('resource-id', '2')).click() // => this.$('[data-test-resource-id="2"]').click()
+this.$(ts`%post-title`).click(); // => this.$('[data-test-post-title]').click();
+this.$(ts`%resource-id=2`).click(); // => this.$('[data-test-resource-id="2"]').click();
 ```
 
```

Comment:
> If you are using a newer version of this addon you are advised to avoid usage of the testSelector function which will be deprecated in the future.

I think we should keep the `testSelector` function and see this as a convenience addition only.